avocado/utils/multipath.py

```diff
@@ -97,6 +97,24 @@ def get_mpath_name(wwid):
         return process.run(cmd, sudo=True).stdout_text.split()[0]
 
 
+def get_mapper_name(dev_mapper):
+    """
+    Get the mpath name for given device mapper
+
+    :param dev_mapper: Input device mapper dmX
```

Comment:
change dmX as dm-X, as the ids are in dm-X format

Reply:
done